notify/config.py

```diff
@@ -69,23 +70,13 @@
         "notify_backends": {
             "type": "object",
             "properties": {
-                "*": {
-                    "type": "object",
-                    "properties": {
-                        "salesforce": {
-                            "type": "object",
-                            "properties": {
-                                "atuh_url": {"type": "string"},
-                                "username": {"type": "string"},
-                                "password": {"type": "string"},
-                                "environment": {"type": "string"},
-                                "client_id":  {"type": "string"},
-                                "client_secret": {"type": "string"},
-                                "organization_id": {"type": "string"}
-                            }
-                        }
-                    }
-                }
+                "*": {"type": "object"}
+            }
+        },
+        "logging": {
```

Comment:
This commit is already merged by another pull request (as dependency) so let's fix this later during integration with oss-lib